custom/icds_reports/ucr/data_sources/cbe_form.json

```diff
@@ -0,0 +1,128 @@
+{
+  "domains": [
+    "icds-dashboard-qa",
+    "reach-sandbox",
+    "reach-test",
+    "icds-sql",
+    "icds-test",
+    "icds-cas",
+    "cas-lab",
+    "icds-cas-sandbox"
+  ],
+  "server_environment": [
+    "softlayer",
+    "icds-new",
+    "icds"
+  ],
+  "config": {
+    "table_id": "static-cbe_form",
+    "display_name": "Forms - Community Based Events",
+    "referenced_doc_type": "XFormInstance",
+    "description": "",
+    "base_item_expression": {},
+    "configured_filter": {
+      "operator": "in",
+      "expression": {
+        "datatype": null,
+        "type": "property_name",
+        "property_name": "xmlns"
+      },
+      "type": "boolean_expression",
+      "property_value": [
+        "http://openrosa.org/formdesigner/61238C23-7059-446D-8A9C-34107642CBB2"
+      ]
+    },
+    "configured_indicators": [
+      {
+        "display_name": "AWC ID",
+        "transform": {},
+        "datatype": "string",
+        "type": "expression",
+        "is_primary_key": false,
+        "is_nullable": true,
+        "column_id": "awc_id",
+        "create_index": true,
+        "expression": {
+          "type": "named",
+          "name": "user_location_id"
+        }
+      },
+      {
+        "display_name": "State ID",
+        "datatype": "string",
+        "expression": {
+          "type": "ancestor_location",
+          "location_id": {
+            "type": "named",
+            "name": "user_location_id"
+          },
+          "location_type": "state",
+          "location_property": "_id"
+        },
+        "is_primary_key": false,
+        "transform": {},
+        "is_nullable": true,
+        "type": "expression",
+        "column_id": "state_id"
+      },
+      {
+        "display_name": null,
+        "datatype": "date",
+        "expression": {
+          "datatype": "date",
+          "type": "property_path",
+          "property_path": [
+            "form",
+            "cbe_info",
+            "date_cbe_organise"
+          ]
+        },
+        "is_primary_key": false,
+        "transform": {},
+        "is_nullable": true,
+        "type": "expression",
+        "column_id": "date_cbe_organise"
+      },
+      {
+        "display_name": null,
+        "transform": {},
+        "datatype": "datetime",
+        "expression": {
+          "expression": {
+            "datatype": "datetime",
+            "type": "property_path",
+            "property_path": [
+              "form",
+              "meta",
+              "timeEnd"
+            ]
+          },
+          "type": "root_doc"
```

Comment:
does this need to be `root_doc`?

Reply:
yeah U are right I dont need that and resolved.

Unrelated to this Pr. root_doc is used to find a tag from the root of the xml doc right? and what is the root_doc then if not form? is it<xml>?